packages/odata-v2/src/request-builder/get-all-request-builder.spec.ts

```diff
@@ -171,7 +171,7 @@ describe('GetAllRequestBuilder', () => {
       const nocks = [
         nock(onlyIssuerXsuaaUrl)
           .post('/oauth/token')
-          .times(2)
+          .times(1)
```

Comment:
yes I optimized the code of the `destination-from-service.ts` so that the clientCrendentials grant to call the destination service is only called once.